components/formats-bsd/src/loci/formats/in/TileJPEGReader.java

```diff
@@ -58,8 +58,7 @@
 
   // -- Fields --
 
-  private JPEGTileDecoder decoder;
-  private JPEGTurboService service;
+  private transient JPEGTurboService service;
```

Comment:
Yeah unfortunately it does, carried out a test locally and it results in:
`2017-07-11 11:24:40,034 [main] WARN  loci.formats.Memoizer - deleting invalid memo file: /Sample Images/jpg/.8kx8k.jpg.bfmemo
com.esotericsoftware.kryo.KryoException: Unable to find class: pg`